packages/auto-utils/src/constants/domain.ts

```diff
@@ -0,0 +1,19 @@
+// file: src/constants/domain.ts
+
+export enum DomainId {
+  AUTO_ID = 'auto-id',
+  NOVA = 'nova',
+}
+
+export const domains = {
+  nova: {
+    id: DomainId.NOVA,
+    name: 'Nova (EVM)',
+    domainId: 0,
```

Comment:
These IDs are network dependent, i.e. you could have a devnet where these are reversed (auto-id:0, nova:1). Would that break this?

Reply:
Yeah, you are right!
So instead of `DomainId` I should call them `DomainRuntime` and only add the id in the networks section... (ounce we know the id as the domain is initiated, and we have rpc)